pkg/kdk/update.go

```diff
@@ -25,19 +26,100 @@ import (
 
 	"github.com/Sirupsen/logrus"
 	"github.com/cisco-sso/kdk/pkg/prompt"
+	"github.com/cisco-sso/kdk/pkg/utils"
+	"github.com/docker/docker/api/types"
+	"github.com/ghodss/yaml"
 	"github.com/mholt/archiver"
 	"github.com/savaki/jq"
 )
 
-func Update(cfg KdkEnvConfig, logger logrus.Entry) error {
-	logger.Info("Updating KDK")
+var (
+	latestReleaseVersion = getLatestReleaseVersion()
+)
 
-	updateImage(cfg, logger)
-	updateBin(logger)
+func Update(cfg KdkEnvConfig, debug bool, skipUpdate bool, logger logrus.Entry) error {
+	if !skipUpdate {
```

Comment:
You could invert the logic here and just return nil early if skipUpdate. after the return you can go into the logical else without saying "else" explicitly.